engines/draci/animation.cpp

```diff
@@ -263,6 +263,11 @@ void Animation::play() {
 	if (isPlaying()) {
 		return;
 	}
+	
+	if (getID() == 671 && _vm->_game->getMapID() == 42) {
```

Comment:
Please add a text comment, what is it, e.g, "Narrator in the Title Screen" or something like that